common/persistence/nosql/nosql_execution_store_util_test.go

```diff
@@ -60,7 +60,7 @@ func TestNosqlExecutionStoreUtils(t *testing.T) {
 						Data:     []byte(`[{"Branches":[{"BranchID":"test-branch-id","BeginNodeID":1,"EndNodeID":2}]}]`),
 					},
 				}
-				return store.prepareCreateWorkflowExecutionRequestWithMaps(workflowSnapshot)
+				return store.prepareCreateWorkflowExecutionRequestWithMaps(workflowSnapshot, time.Now())
```

Comment:
Have you had any error when running the unit tests? I was thinking to use a constant time stamp in the tests like [this ](https://github.com/cadence-workflow/cadence/blob/e57792ddb62cf10fd51bb3c13606da33fea12c72/common/persistence/nosql/nosqlplugin/cassandra/workflow_utils_test.go#L45) so that in the unit tests it will always keep the same

Reply:
No, there haven't been any errors in the tests with this implementation. However, I refactored it to use a FixedTime variable for consistency.